community/import-tool/src/docs/ops/import-tool.asciidoc

```diff
@@ -382,11 +384,50 @@ The call to `neo4j-import` would look like this:
 include::same-relationship-type-everywhere.adoc[]
 ----
 
-[[import-tool-working-with-sequential-ids]]
+[[import-tool-property-types]]
+=== Property Types
+
+The type for properties specified in nodes and relationships files is defined in the header row.
+(see <<import-tool-header-format>>)
+
+The following example creates a small graph containing one actor and one movie connected by an `ACTED_IN` relationship.
+There is a `roles` property on the relationship which contains an array of the characters played by the actor in a movie.
+
+.movies10.csv
+[source]
+----
+include::movies10.csv[]
+----
+
+.actors10.csv
+[source]
+----
+include::actors10.csv[]
+----
+
+.roles10.csv
+[source]
+----
+include::roles10.csv[]
+----
+
+The arguments to `neo4j-import` would be the following:
+
+[source]
+----
+include::property-types.adoc[]
+----
+
+[[import-tool-id-handling]]
+== ID handling
+
+Each node processed by `neoj4-import` must provide a unique id.
+We use this id to find the correct nodes when creating relationships.
+
 === Working with sequential or auto incrementing identifiers
 
 The import tool makes the assumption that identifiers are unique across node files.
-This may not be the case for data sets which use sequential or auto incremented primary keys.
+This may not be the case for data sets which use sequential, auto incremented or otherwise colliding identifiers.
 For those data sets we can define an `id space` in which our identifier is unique.
```

Comment:
The use of `our` strikes me as odd here. Also `an id space` implies a single id space, better with `id spaces`.

A suggestion: `Those data sets can define `id spaces` where identifiers are unique within their respective id space`. Not super happy about that sentence, but something in that direction?